src/screens/Auth/RegisterStack/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -21,11 +21,13 @@ export default function CreatePasswordScreen(props: Props) {
   const [confirmPassword, setConfirmPassword] = useState('')
   const {navigate} = props.navigation 
   const user = {
```

Comment:
can you insert a variable above this 
`const {params} = props.navigation.state.params then refer to it below since it's so repetitive

Reply:
I tried that but it always gives an error saying undefined is not an object for 'params.companyCode' and the other fields that use it?

Reply:
ok fuck it nvm